app/src/test/java/org/simple/clinic/security/pin/BruteForceProtectionStateTest.kt

```diff
@@ -0,0 +1,29 @@
+package org.simple.clinic.security.pin
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.di.NetworkModule
+import org.simple.clinic.util.Just
+import org.threeten.bp.Instant
+
+class BruteForceProtectionStateTest {
+
+  /**
+   * If you're seeing this test fail, it means that you modified
+   * [BruteForceProtectionState] without thinking about migration.
```

Comment:
We also need to plan as to how exactly we **do** migration for shared preferences, if ever required.

Reply:
Done